src/Typed/assets/package.json

```diff
@@ -2,7 +2,16 @@
     "name": "@symfony/ux-typed",
     "description": "Typed integration for Symfony",
     "license": "MIT",
-    "version": "1.0.0",
+    "version": "2.23.0",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://ux.symfony.com/typed",
+    "repository": "https://github.com/symfony/ux/tree/2.x/src/Typed",
+    "type": "module",
+    "files": [
+        "dist"
+    ],
```

Comment:
Same comment than Swipe .... but i would 100% understand if the simpler for you here is to do them all the same way